README.md

```diff
@@ -80,26 +80,6 @@ In the pull request where you previously ran `bot queue`, comment:
 The API provides an alternative interface for executing commands directly
 without having to go through pull request comments.
 
-## Create a Personal Token <a name="api-create-token"></a>
-
-For interacting with the commands, first a Personal Token needs to be generated
-through `POST /api/access` by the
-[`$MASTER_TOKEN`](#setup-environment-variables) (it is currently owned
-by the OpsTooling team of Parity for Parity's deployments).
-
-Each Personal Token is tied to a Matrix Room ID where the command's outcome will
-be posted to after it finishes.
-
-```sh
-curl \
-  -H "X-Auth: $MASTER_TOKEN" \
-  -H "Content-Type: application/json" \
-  -X POST http://command-bot/access \
-  -d '{
-    "matrixRoom": "!tZrvvMzoIkIYbCkLuk:matrix.foo.io",
-  }'
-```
-
```

Comment:
Does this imply that we should kill the API altogether, as there's no way to get the token anymore?

Reply:
no, this still checks mastertoken

Reply:
ahh, I see
in that case, let's replace "personal token" with "master token" in the next section, and also let's update TOC

Reply:
right, done, thx!